06-reactjs/components/movie-row.js

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
```

Comment:
I'm not pretty sure if this component is needed. You're only displaying information, no logic or events are defined here. What do you think @troianoandres?